static/api/admin-api.yaml

```
@@ -0,0 +1,4516 @@
+openapi: 3.0.1
+info:
+  title: Redpanda Admin API
+  version: 1.0.0
+  description: |
+  
+    Use the Admin API to administer and manage Redpanda.
+
+    ---
+
+servers:
+- url: http://localhost:9644/v1
+paths:
+  # /config:
+  #   get:
+  #     tags:
+  #     - Cluster
+  #     summary: Get cluster configurations
+  #     description: |
+  #       Get the cluster configuration properties of a cluster.
+
+  #       For supported properties, see [/cluster_config/schema](#operation/get_cluster_config_schema).
+  #     operationId: get_config
+  #     responses:
+  #       200:
+  #         description: Cluster configuration response
+  #         content:
+  #           application/json:
+  #               schema:
+  #                 type: array
+  #                 items:
+  #                   $ref: '#/components/schemas/cluster_config_properties'
+  /node_config:
+    get:
+      tags:
+      - Clusters
+      summary: Get brokers configurations
+      description: |
+        List the configuration properties of the nodes (brokers) in the cluster.
```

```suggestion
        List the configuration properties of all brokers (nodes) in the cluster.
```

We should be consistent with what we put in brackets. Later in this doc we use `brokers (nodes)`.